[stable/airflow] fix postgres database config name

thesuperzapper opened the pull request:
The docs suggest that `postgresql.postgresDatabase` was renamed to `postgresql.postgresqlDatabase`, this hotfix makes that true.

gsemet reviewed (commented):
You have to update the chart version (and maybe rebase)

thesuperzapper committed:
[stable/airflow] fix postgres database config name

The docs suggest that `postgresql.postgresDatabase` was renamed to `postgresql.postgresqlDatabase`, this hotfix makes that true.

Signed-off-by: Mathew Wicks <mathew.wicks@gmail.com>
